package.json

```
@@ -24,14 +25,15 @@
 		"typescript-eslint": "^8.32.1"
 	},
 	"scripts": {
-		"dev:web": "bun --cwd=apps/web dev",
-		"dev:server": "bun --cwd=apps/server dev",
-		"dev": "concurrently --names \"WEB,SERVER\" --prefix-colors \"cyan,magenta\" --prefix \"[{name}]\" --kill-others-on-fail \"bun --cwd=apps/web dev\" \"bun --cwd=apps/server dev\"",
+		"dev:web": "bun --elide-lines 0 --filter @nimbus/web --env-file .env dev",
+		"dev:server": "bun --elide-lines 0 --filter @nimbus/server --env-file .env dev",
+		"dev": "concurrently --names \"WEB,SERVER\" --prefix-colors \"cyan,magenta\" --prefix \"[{name}]\" --kill-others-on-fail \"bun --filter * --env-file .env dev\"",
+		"dev:verbose": "bun --elide-lines 0 --filter \"*\" --env-file .env dev",
 		"db:up": "docker compose up -d",
 		"db:down": "docker compose down",
 		"db:reset": "docker compose down -v && docker compose up -d",
-		"db:migrate": "cd packages/db && bunx drizzle-kit generate && bunx drizzle-kit migrate",
-		"db:studio": "cd packages/db && bunx drizzle-kit studio",
+		"db:migrate": "bun --filter @nimbus/db migrate",
+		"db:studio": "bun --filter @nimbus/db studio",
```

```suggestion
		"db:migrate": "bun --elide-lines 0 --filter @nimbus/db --env-file .env migrate",
		"db:studio": "bun --elide-lines 0 --filter @nimbus/db --env-file .env studio",
```
I recommend adding this so the config can have access to the `DATABASE_URL` variable